src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpBaseStream.cs

```diff
@@ -103,8 +103,7 @@ public override void Write(byte[] buffer, int offset, int count)
             // situations.  Either a derived stream overrides this anyway, so the implementation won't be used,
             // or it's being called as part of HttpContent.SerializeToStreamAsync, which means custom
             // content is explicitly choosing to make a synchronous call as part of an asynchronous method.
-            ValidateBufferArgs(buffer, offset, count);
-            WriteAsync(new Memory<byte>(buffer, offset, count), CancellationToken.None).GetAwaiter().GetResult();
+            WriteAsync(buffer, offset, count, default).GetAwaiter().GetResult();
```

Comment:
I see we're passing in `default` instead of `CancellationToken.None`. Is the former a better pattern?

Reply:
No, they're both identical.  My fingers just typed the former :)  I can change it.